framework/PageInputs/PageAsyncMultiSelect.tsx

```diff
@@ -25,6 +31,9 @@ export interface PageAsyncMultiSelectProps<ValueT>
 
   /** The placeholder to show if the query fails. */
   queryErrorText?: string | ((error: Error) => string);
+
+  /** The function to open the browse modal. */
+  openBrowse?: PageOpenMultiSelectBrowse<ValueT>;
```

Comment:
I'd like to keep the browse button logic out of the PageAsyncMultiSelect.
PageAsyncMultiSelect works fine with dynamically loading items in it's current state.
Trying to keep PageAsyncMultiSelect as simple as possible.

For any places that use PageAsyncMultiSelect and want to support a 'browse',
It can be passed in as the 'footer' prop to the PageAsyncMultiSelect.
Might be good to have a PageSelectBrowseFooter component that encapsulates that support.

Makes the components more composable.

Reply:
Done, but I choose to implement it inside the component params directly, no need to move it to shared functions, since we would need to pass half of parameters anyway (default selection, on select) only to render button :)